### DISCUSSION TITLE
How I can keep a model in gpu memory?

### DISCUSSION THREAD

**Discussion by mliradelc** [openai/whisper#574] [Q&A] [answered]
Hi. 
Thanks so much Open AI for make this project open!, I started to play with it since 2 months ago and it's amazing.

Now I have a question. Every time I run whisper, this loads the model to the GPU memory, I want to know if it's possible to keep the model always in memory so it will be faster.

Best regards.

Max

**Comment by jongwook** [comment, marked as answer]
Hi, you can specify multiple audio files in the command line like `whisper *.mp3` to load the model once and transcribe all files. For more control, you'll need to use the Python interface for this because the GPU memory is released once the Python process exits. Please also see #516.

**Comment by mliradelc** [comment]
Thank you so much @jongwook for your answer, now I understand better why some other whisper projects can keep the models loaded!
